### PR TITLE
Plot storage in project

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**

### CODE REVIEW

**Comment by ChristianBeilschmidt** [comment]
[Plot storage in project](https://trello.com/c/Kgxxuzpy/312-plot-storage-in-project)

File: services/src/projects/postgres_projectdb.rs [hunk 1]
```diff
@@ -40,6 +42,96 @@ where
     pub fn new(conn_pool: Pool<PostgresConnectionManager<Tls>>) -> Self {
         Self { conn_pool }
     }
+
+    async fn list_plots(
```
Comment (michaelmattig) [inline services/src/projects/postgres_projectdb.rs:46]: validate permission via `PostgresContext::check_user_project_permission`

File: services/src/projects/postgres_projectdb.rs [hunk 2]
```diff
@@ -40,6 +42,96 @@ where
...
+            )
+            .await?;
+
+        let plot_rows = conn.query(&stmt, &[project_version_id]).await?;
+        let plot_names = plot_rows.iter().map(|row| row.get(0)).collect();
+
+        Ok(plot_names)
+    }
+
+    async fn load_plots(
```
Comment (michaelmattig) [inline services/src/projects/postgres_projectdb.rs:67]: validate permission via `PostgresContext::check_user_project_permission`

File: services/src/projects/postgres_projectdb.rs [hunk 3]
```diff
@@ -40,6 +42,96 @@ where
...
+            .map(|row| Plot {
+                workflow: WorkflowId(row.get(2)),
+                name: row.get(1),
+            })
+            .collect();
+
+        Ok(plots)
+    }
+
+    async fn update_plots(
```
Comment (michaelmattig) [inline services/src/projects/postgres_projectdb.rs:97]: validate permission via `PostgresContext::check_user_project_permission`

File: services/src/projects/postgres_projectdb.rs (hunk 1 above)
Comment (ChristianBeilschmidt) [reply services/src/projects/postgres_projectdb.rs:46]: checked in list

File: services/src/projects/postgres_projectdb.rs (hunk 2 above)
Comment (ChristianBeilschmidt) [reply services/src/projects/postgres_projectdb.rs:67]: checked in load

File: services/src/projects/postgres_projectdb.rs (hunk 3 above)
Comment (ChristianBeilschmidt) [reply services/src/projects/postgres_projectdb.rs:97]: checked in update

**Pull request merged**